src/time.rs

```diff
@@ -542,12 +572,13 @@ impl TryFrom<&str> for MicrosecondsPrecisionOverflowBehavior {
         match value.to_lowercase().as_str() {
             "truncate" => Ok(Self::Truncate),
             "error" => Ok(Self::Error),
-            _ => Err(ConfigError::UnknownSecondsPrecisionOverflowBehaviorString),
+            _ => Err(ConfigError::UnknownMicrosecondsPrecisionOverflowBehaviorString),
         }
     }
 }
 
 #[derive(Debug, Clone, Default)]
 pub struct TimeConfig {
     pub microseconds_precision_overflow_behavior: MicrosecondsPrecisionOverflowBehavior,
+    pub unix_timestamp_offset: Option<i32>,
```

Comment:
I propose that the default should be `Some(0)` (you'll have to implement `Default` manually for this to be the case unfortunately).

The reasoning: the unix timestamp is defined as UTC+0, so I think it is correct to treat timestamps as timezone-aware by default.

Reply:
I chose `None` because that's the current behavior of `speedate` and we can choose a different default in `pydantic-core`, i.e. by using `None` we don't break any current users of `speedate`. Also It's less code 😝